sensor/kubernetes/sensor/sensor.go

```diff
@@ -227,6 +231,7 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 		complianceService,
 		imageService,
 		deployment.NewService(storeProvider.Deployments(), storeProvider.Pods()),
+		collectorRuntimeConfigService,
```

Comment:
Perhaps this is where the feature flag should be used?

Reply:
Yes! But I would also add one for adding the component to `components := []common.SensorComponent{`

Reply:
I have moved the check for the feature flag here and to the other place that you suggested.